scripts/eosio_build_ubuntu.sh

```diff
@@ -62,7 +62,7 @@
 
 	DEP_ARRAY=(clang-4.0 lldb-4.0 libclang-4.0-dev cmake make automake libbz2-dev libssl-dev \
 	libgmp3-dev autotools-dev build-essential libicu-dev python2.7-dev python3-dev \
-    autoconf libtool curl zlib1g-dev doxygen graphviz)
+    autoconf libtool curl zlib1g-dev doxygen graphviz libsoci-dev)
```

Comment:
This dependency should be installed by the plugin build process, not by the eosio build process.